server/storage/mvcc/key_index.go

```diff
@@ -329,6 +329,8 @@ func (ki *keyIndex) equal(b *keyIndex) bool {
 
 func (ki *keyIndex) String() string {
 	var s string
+	s += string(ki.key[:]) + "\n"
```

Comment:
After second thought, the len(ki.key) might be very large, so it may pollute the output. I would suggest to truncate it and add "..." if it exceeds a threshold, e.g. 128 bytes.

Reply:
update, PTAL